airflow/providers/snowflake/operators/snowflake.py

```diff
@@ -131,281 +126,3 @@ def execute(self, context: Any) -> None:
 
         if self.do_xcom_push:
             return execution_info
-
-
-class SnowflakeCheckOperator(SQLCheckOperator):
```

Comment:
Why did we remove this operator

Reply:
This PR was opened as an alternative to adding more `SnowflakeCheckOperators`, so it seemed to me that their removal here made sense given the nature of this change making all these `*Check*` operators redundant. Although I see how it could warrant a separate PR.

Reply:
Yes a separate PR and we don't want the current users of this Operator to have to change their DAGs so let's not remove them please